lib/datawow/connector.rb

```diff
@@ -0,0 +1,93 @@
+# frozen_string_literal: true
+
+require 'net/http'
+require 'net/https'
+require 'uri'
+
+require File.expand_path('client_response.rb', __dir__)
+
+module Datawow
```

Comment:
Please move code prepare uri and header to method
ex. https://github.com/omise/omise-ruby/blob/master/lib/omise/resource.rb

Add basic http verb GET, POST, PATCH, DELETE